readme.md

```diff
@@ -224,19 +237,18 @@ var handleClickEvent = function(){
 button.on("click", handleClickEvent);
 ```
 
-Our favorite keyword: `this`.
-* **Q:** Where have we seen it before? Why do we use it?  
-
 Try this: insert the following line anywhere in our event listener...
 * `console.log($(this))`
 * Now what do you see when we click the button? How would you define `this` in the context of an event listener?
 
 In the context of an event listener callback, `$(this)` always refers to the object that triggered the event.
 
-### You Do: `this` Practice (10 minutes / 1:35)
+### You Do: `this` Practice (4:05 - 4:15, 10 minutes)
 
 Clone and follow the instructions in this [repository](https://github.com/ga-wdi-exercises/events-this-practice).
 
+#### Bonus: `this` and Color Scheme Picker
+
 > If you're curious, here's a short solution to the earlier Color Scheme Picker exercise that makes use of `this`. Keep in mind, this is advanced! You are not expected to be able to write code like this yet.  
```

Comment:
maybe wrap this part in a details, especially if you decide to do color scheme switcher instead